config/get_started.json

```diff
@@ -22,56 +22,90 @@
         "only": false,
         "countries": ["us", "tw"]
     },
-    "pii_map": {
+    "credentialDefinition": "employee.cred.def.json",
+    "credentialKeys": {
         "firstName": {
             "name": "First Name",
-            "dataType": "text"
+            "dataType": "text",
+            "credentials": {
+                "employee.cred.def.json": true
```

Comment:
This repetition of `employee.cred.def.json` suggests reusing the `CredentialKeys` among  different credentials versus overlaying each credential definition with its own metadata. This give me pause because it will be good to have single credential overlay we can share among agents (including mobile).

Reply:
I went back and forth on this, but given that we're creating the `pii_map` at build-time it's probably ok to just have, like:
```
{
  "employee.cred.def.json": {
    "firstName": {...}
  },
  "borrower.cred.def.json": {
    "firstName": {...}
  }
}
```

Ultimately I'd love this information to live in its own JSON file and be handled separately for easier handling of multiple credential types